### ISSUE TITLE
OpenCL.dll is missing on Win 10 x64

### ISSUE THREAD

**Issue by Nickbert7** [jellyfin/jellyfin-ffmpeg#81]
I'm not sure if the issue is right here or should be moved to the server repo.
Today I wanted to test the latest unstable build for windows [20220115.12](https://repo.jellyfin.org/releases/server/windows/versions/unstable/combined/20220115.12/)
During the startup I get two errors telling me that the OpenCL.dll was not found:
![image](https://user-images.githubusercontent.com/14938497/149622675-ddb8eddc-aa64-4e6a-96ac-c557f2c46685.png)

Logs:
````[2022-01-15 14:12:41.426 +01:00] [INF] [7] Main: Jellyfin version: "10.8.0"
[2022-01-15 14:12:41.498 +01:00] [INF] [7] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, D:\Server\jellyfin-test\data\log]"]
[2022-01-15 14:12:41.535 +01:00] [INF] [7] Main: Arguments: ["D:\Server\jellyfin-test\system\jellyfin.dll", "-d", "D:\Server\jellyfin-test\data"]
[2022-01-15 14:12:41.546 +01:00] [INF] [7] Main: Operating system: "Windows"
[2022-01-15 14:12:41.549 +01:00] [INF] [7] Main: Architecture: X64
[2022-01-15 14:12:41.555 +01:00] [INF] [7] Main: 64-Bit Process: True
[2022-01-15 14:12:41.572 +01:00] [INF] [7] Main: User Interactive: True
[2022-01-15 14:12:41.576 +01:00] [INF] [7] Main: Processor count: 4
[2022-01-15 14:12:41.580 +01:00] [INF] [7] Main: Program data path: "D:\Server\jellyfin-test\data"
[2022-01-15 14:12:41.583 +01:00] [INF] [7] Main: Web resources path: "D:\Server\jellyfin-test\system\jellyfin-web"
[2022-01-15 14:12:41.585 +01:00] [INF] [7] Main: Application directory: "D:\Server\jellyfin-test\system\"
[2022-01-15 14:12:42.268 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Marking following migrations as applied because this is a fresh install: ["CreateNetworkConfiguration"]
[2022-01-15 14:12:43.128 +01:00] [INF] [7] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "D:\Server\jellyfin-test\data\cache"
[2022-01-15 14:12:43.878 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[2022-01-15 14:12:48.630 +01:00] [INF] [7] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses : "[10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2022-01-15 14:12:48.634 +01:00] [INF] [7] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions : "[]"
[2022-01-15 14:12:48.639 +01:00] [INF] [7] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: "[10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]"
[2022-01-15 14:12:48.674 +01:00] [INF] [7] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: "[]"
[2022-01-15 14:12:48.678 +01:00] [INF] [7] Jellyfin.Networking.Manager.NetworkManager: Using bind exclusions: "[]"
[2022-01-15 14:13:09.667 +01:00] [INF] [7] Jellyfin.Server.Implementations.JellyfinDbProvider: There are pending EFCore migrations in the database. Applying... (This may take a while, do not stop Jellyfin)
[2022-01-15 14:13:10.726 +01:00] [INF] [7] Jellyfin.Server.Implementations.JellyfinDbProvider: EFCore migrations applied successfully
[2022-01-15 14:13:12.216 +01:00] [INF] [7] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[2022-01-15 14:13:12.225 +01:00] [INF] [7] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "D:\Server\jellyfin-test\data\cache"
[2022-01-15 14:13:12.257 +01:00] [INF] [7] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: "TMDb" "10.8.0.0"
[2022-01-15 14:13:12.274 +01:00] [INF] [7] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: "Studio Images" "10.8.0.0"
[2022-01-15 14:13:12.287 +01:00] [INF] [7] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: "OMDb" "10.8.0.0"
[2022-01-15 14:13:12.297 +01:00] [INF] [7] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: "MusicBrainz" "10.8.0.0"
[2022-01-15 14:13:12.312 +01:00] [INF] [7] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: "AudioDB" "10.8.0.0"
[2022-01-15 14:13:12.731 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Marking following migrations as applied because this is a fresh install: ["DisableTranscodingThrottling", "CreateLoggingConfigHeirarchy", "MigrateActivityLogDatabase", "RemoveDuplicateExtras", "MigrateUserDatabase", "MigrateDisplayPreferencesDatabase", "RemoveDownloadImagesInAdvance", "MigrateAuthenticationDatabase"]
[2022-01-15 14:13:12.743 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Applying migration '"AddDefaultPluginRepository"'
[2022-01-15 14:13:12.750 +01:00] [INF] [7] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[2022-01-15 14:13:12.755 +01:00] [INF] [7] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "D:\Server\jellyfin-test\data\cache"
[2022-01-15 14:13:12.762 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Migration '"AddDefaultPluginRepository"' applied successfully
[2022-01-15 14:13:12.796 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Applying migration '"ReaddDefaultPluginRepository"'
[2022-01-15 14:13:12.800 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Migration '"ReaddDefaultPluginRepository"' applied successfully
[2022-01-15 14:13:12.808 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Applying migration '"AddPeopleQueryIndex"'
[2022-01-15 14:13:12.816 +01:00] [INF] [7] Jellyfin.Server.Migrations.Routines.AddPeopleQueryIndex: Creating index idx_TypedBaseItemsUserDataKeyType
[2022-01-15 14:13:12.827 +01:00] [INF] [7] Jellyfin.Server.Migrations.Routines.AddPeopleQueryIndex: Creating index idx_PeopleNameListOrder
[2022-01-15 14:13:12.842 +01:00] [INF] [7] Jellyfin.Server.Migrations.MigrationRunner: Migration '"AddPeopleQueryIndex"' applied successfully
[2022-01-15 14:13:12.889 +01:00] [INF] [7] Main: Kestrel listening on "Any IP4 Address"
[2022-01-15 14:13:25.767 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: Running startup tasks
[2022-01-15 14:13:26.000 +01:00] [INF] [7] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for "Extract Chapter Images" set to fire at 2022-01-16 02:00:00.000 +01:00, which is 11:46:34.0002971 from now.
[2022-01-15 14:13:29.032 +01:00] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: "StartupTrigger" fired for task: "Update Plugins"
[2022-01-15 14:13:29.038 +01:00] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task "PluginUpdateTask"
[2022-01-15 14:13:29.067 +01:00] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Update Plugins"
[2022-01-15 14:13:30.124 +01:00] [ERR] [7] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg validation: The process returned no result
[2022-01-15 14:13:30.128 +01:00] [WRN] [7] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: Failed version check: "ffmpeg"
[2022-01-15 14:13:30.181 +01:00] [INF] [7] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: ""
[2022-01-15 14:13:30.192 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: ServerId: "4164eb5ca030473283da9543aea4dbf8"
[2022-01-15 14:13:30.799 +01:00] [INF] [18] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Update Plugins" Completed after 0 minute(s) and 1 seconds
[2022-01-15 14:13:30.944 +01:00] [INF] [18] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[2022-01-15 14:13:31.045 +01:00] [INF] [7] Emby.Dlna.Main.DlnaEntryPoint: Registering publisher for "urn:schemas-upnp-org:device:MediaServer:1" on "192.168.1.5/24"
[2022-01-15 14:13:31.086 +01:00] [INF] [7] Emby.Dlna.Main.DlnaEntryPoint: Registering publisher for "urn:schemas-upnp-org:device:MediaServer:1" on "127.0.0.1/8"
[2022-01-15 14:13:31.101 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: Executed all pre-startup entry points in 0:00:00.8763864
[2022-01-15 14:13:31.104 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: Core startup complete
[2022-01-15 14:13:31.737 +01:00] [INF] [7] Emby.Server.Implementations.ApplicationHost: Executed all post-startup entry points in 0:00:00.6293803
[2022-01-15 14:13:31.743 +01:00] [INF] [7] Main: Startup complete 0:00:54.5832505
[2022-01-15 14:13:33.465 +01:00] [WRN] [4] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.1.5:8096/dlna/4164eb5ca030473283da9543aea4dbf8/description.xml" to "192.168.1.73" in 0:00:00.9706798 with Status Code 200
````

**Comment by nyanmisaka** [comment]
Can you install your graphics driver or this opencl runtime? https://www.microsoft.com/en-us/p/opencl-and-opengl-compatibility-pack/9nqpsl29bfff

**Comment by Nickbert7** [comment]
The installation of both did not solve the error.
But is seems OpenCL is generally not supported by my (quite old) CPU/GPU: Intel Core i3-350M

**Comment by nyanmisaka** [comment]
This issue will be fixed by #82. Thanks for your feedback!

You can replace your ffmpeg with the latest build here. https://github.com/jellyfin/jellyfin-ffmpeg/suites/4952229870/artifacts/143767632
